.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.6
```

Comment:
What is this for? 

I understand the purpose of the file. But I am wondering why you need Ruby in this project?

Reply:
Because [React Native uses Ruby in some scripts related to the iOS dependency management](https://reactnative.dev/docs/environment-setup#ruby) and requires it for running.